aws/v12.1.1/release.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v12.1.1
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.2.3
+  - componentVersion: 0.15.2
+    name: cert-manager
+    version: 2.0.2
+  - name: chart-operator
+    version: 0.13.2
+  - componentVersion: 1.17.3
+    name: cluster-autoscaler
+    version: 1.17.3
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.2.0
+  - componentVersion: 0.7.2
+    name: external-dns
+    version: 1.2.2
+  - componentVersion: 3.6.0
+    name: kiam
+    version: 1.4.0
+  - componentVersion: 1.9.7
+    name: kube-state-metrics
+    version: 1.1.1
+  - componentVersion: 0.3.6
+    name: metrics-server
+    version: 1.1.1
+  - name: net-exporter
+    version: 1.9.0
+  - componentVersion: 1.0.1
+    name: node-exporter
+    version: 1.3.0
+  components:
+  - name: app-operator
+    version: 1.0.0
+  - name: aws-cni
+    version: 1.6.3
+  - name: aws-operator
+    releaseOperatorDeploy: true
+    version: 8.8.0
+  - name: calico
+    version: 3.15.1
+  - name: cert-operator
+    releaseOperatorDeploy: true
+    version: 0.1.0
+  - name: cluster-operator
+    releaseOperatorDeploy: true
+    version: 2.3.2
+  - name: containerlinux
+    version: 2512.2.1
+  - name: etcd
+    version: 3.4.10
+  - name: kubernetes
+    version: 1.17.9
+  date: "2020-08-12T12:00:00Z"
```

Comment:
TODO: Update once more when we know when to release

Reply:
true